stories/examples/controlled.js

```diff
@@ -34,6 +34,13 @@ class Examples extends Component {
   items = ['Black', 'Red', 'Green', 'Blue', 'Orange', 'Purple']
 
   changeHandler = changes => {
+    this.setState({
+      selectedColor: changes.selectedValue,
+      isOpen: false,
```

Comment:
This is so now when you select a value, the controller (the user of the downshift component) will close the menu, which is neat.